diff -urN scigraphica-0.8.0/src/Makefile.am scigraphica-0.8.0.patch/src/Makefile.am
--- scigraphica-0.8.0/src/Makefile.am	Tue Nov 13 07:39:08 2001
+++ scigraphica-0.8.0.patch/src/Makefile.am	Wed Dec 11 23:06:22 2002
@@ -187,3 +187,6 @@
             echo $$subdir/$$file; \
           done; \
         done
+
+python/sg.so:
+	$(CC) $(LIBS) $(GNOME_PRINT_LIBS) $(BONOBO_LIBS) -shared *.o python/*.a ../zvt/*.a -o python/sg.so
diff -urN scigraphica-0.8.0/src/python/python_int.c scigraphica-0.8.0.patch/src/python/python_int.c
--- scigraphica-0.8.0/src/python/python_int.c	Fri Nov 30 18:03:23 2001
+++ scigraphica-0.8.0.patch/src/python/python_int.c	Wed Dec 11 23:04:52 2002
@@ -105,7 +105,7 @@
 
 
 /* sg module init */
-    init_sg_python();
+    initsg();
     sg_o=PyImport_AddModule("sg");
     sg_dict=PyModule_GetDict(sg_o);
     PyDict_SetItemString (main_dict, "sg", sg_o);  
diff -urN scigraphica-0.8.0/src/python/python_sheet.c scigraphica-0.8.0.patch/src/python/python_sheet.c
--- scigraphica-0.8.0/src/python/python_sheet.c	Fri Nov 30 18:03:23 2001
+++ scigraphica-0.8.0.patch/src/python/python_sheet.c	Wed Dec 11 23:05:09 2002
@@ -1566,7 +1566,7 @@
   {NULL, NULL, 0,NULL}        /* Sentinel */
 };
 
-void init_sg_python()
+void initsg()
 { 
   (void) Py_InitModule("sg", SGMethods);
 }
